src/main/java/uk/gov/hmcts/reform/sendletter/tasks/SerialTaskRunner.java

```diff
@@ -97,8 +95,9 @@ private boolean executeReturningBool(Connection connection, String sql) throws S
         try (ResultSet set = connection.createStatement().executeQuery(sql)) {
             if (set.next()) {
                 return set.getBoolean(1);
+            } else {
+                return false;
```

Comment:
now this block looks a lot like simple return statement: `return set.next();`

Reply:
not really, we `.getBoolean()` may still be have `false` value

Reply:
oh yea, where am i looking¿